### DISCUSSION TITLE
出外靠朋友，DARPA 將為醫護兵和工程師開發 AI 輔助器材 AMIGOS

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#179] [General]
雖然 AMIGOS 不能完全取代訓練有素的醫生或工程師，但戰爭期間很多情況下這些專家也無法隨傳隨到，因此設備期望藉由 AI 技術，讓即使沒有相關訓練的人員也能緊急簡單醫護和維修，大幅降低後勤負擔和人員裝備損失率。

https://technews.tw/2022/01/04/darpa-is-developing-amigos-which-can-help-medics-and-mechanics-operate-unfamiliar-procedures-with-ai-and-ar-technologies/

**Comment by briankuo123** [comment]
我覺得以AI為輔助的工具是日後AI發展的主軸，有許多東西都可以靠AI幫助來更進一步去發展。

**Comment by minniewei** [comment]
但同時一定會很多人感到害怕怕有朝一日ai取代他的工作

**Comment by wh20502** [comment]
我覺得擴增實境可以增加滿多方便性的，如果用在運動上的話應該也很有趣，可以偵測球速阿什麼的，但是現在的裝置都還太大。

**Comment by Lynn12378** [comment]
感覺很有幫助，運用在其他地方也很有用!
